R/settings_quarto_website.R

```diff
@@ -15,18 +15,30 @@
     settings <- gsub(": no$", ": false", settings)
     writeLines(settings, fn)
 
-    # copy README.md to _quarto/index.qmd
-    src <- fs::path_join(c(path, "README.qmd"))
-    tar <- fs::path_join(c(path, "_quarto", "index.qmd"))
-    fs::file_copy(src, tar, overwrite = TRUE)
+    # copy README to _quarto/index.qmd
+    src_qmd <- fs::path_join(c(path, "README.qmd"))
+    src_rmd <- fs::path_join(c(path, "README.Rmd"))
+    src_md <- fs::path_join(c(path, "README.md"))
+    if (fs::file_exists(src_qmd)) {
+        tar <- fs::path_join(c(path, "_quarto", "index.qmd"))
+        fs::file_copy(src_qmd, tar, overwrite = TRUE)
+    } else if (fs::file_exists(src_qmd)) {
```

Comment:
this is the same condition as in the first `if`, I guess one of the two should be `src_md`

Reply:
thanks, good catch